packages/design-system-react/src/types/index.ts

```diff
@@ -142,6 +142,53 @@ export type BadgeWrapperCustomPosition = {
   left?: number | string | undefined;
 };
 
+/**
+ * Box - all spacing-related props
+ */
+export type BoxSpacing = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
+/**
+ * Box - flexDirection
+ */
+export enum BoxFlexDirection {
+  Row = 'flex-row',
+  RowReverse = 'flex-row-reverse',
+  Column = 'flex-col',
+  ColumnReverse = 'flex-col-reverse',
+}
+
+/**
+ * Box - flexWrap
+ */
+export enum BoxFlexWrap {
+  NoWrap = 'flex-nowrap',
+  Wrap = 'flex-wrap',
+  WrapReverse = 'flex-wrap-reverse',
+}
+
+/**
+ * Box - alignItems
+ */
+export enum BoxAlignItems {
+  Start = 'items-start',
+  Center = 'items-center',
+  End = 'items-end',
+  Stretch = 'items-stretch',
+  Baseline = 'items-baseline',
+}
+
+/**
+ * Box - justifyContent
+ */
+export enum BoxJustifyContent {
+  Start = 'justify-start',
+  Center = 'justify-center',
+  End = 'justify-end',
+  Between = 'justify-between',
+  Around = 'justify-around',
+  Evenly = 'justify-evenly',
+}
```

Comment:
Checking against extension we are supporting each but using a different name. I think this is a reasonably breaking change

```
export enum JustifyContent {
  flexStart = 'flex-start',
  flexEnd = 'flex-end',
  center = 'center',
  spaceAround = 'space-around',
  spaceBetween = 'space-between',
  spaceEvenly = 'space-evenly',
}
```

https://github.com/MetaMask/metamask-extension/blob/main/ui/helpers/constants/design-system.ts#L270-L277